frontend/test/metabase/scenarios/binning/correctness/time-series.cy.spec.js

```diff
@@ -168,11 +169,11 @@ function getVisualization(binningType) {
 }
 
 function assertOnXYAxisLabels() {
-  cy.get(".y-axis-label")
+  cy.get(".y-axis-label", { timeout: 10000 })
```

Comment:
Increasing the timeout here is rather dreadful, but I'm running out of tricks on how to wait until chart construction finishes, particularly on those low-spec CI runner(s).

Reply:
What if we take an alternative approach here and limit the number of results to something smaller.
It should solve this problem and should also dramatically speed up the whole `binning` e2e group.

I can work on that tomorrow if it sounds like a good idea?

Reply:
Yup, probably worth giving it a shot @nemanjaglumac!

Reply:
Limiting the number of results changes the visualization to table. That shouldn't invalidate the thing we're testing here, right?